modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -363,6 +384,9 @@ func (h *Handler) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 			if h.HealthChecks.Active == nil {
 				h.HealthChecks.Active = new(ActiveHealthChecks)
 			}
+			if h.HealthChecks.Active.Upstream != "" {
+				caddy.Log().Named("config.adapter.caddyfile").Warn("the 'health_port' subdirective is ignored if 'health_upstream' is used!")
+			}
```

Comment:
It might be worth adding more than just a log here to inform that the `health_port` is being ignored.

Reply:
Yeah we could `return d.Errf` instead of a warning